gh-skeleton

```diff
@@ -0,0 +1,254 @@
+
+  log_info "Creating ${dest_org}/${dest_repo} from skeleton ${src_org}/${src_repo}."
+
+  log_info "Cloning skeleton remote repository to the new local repository."
+  git clone --origin "${src_repo}" git@github.com:"${src_org}/${src_repo}".git "${dest_repo}"
+
+  cd "${dest_repo}"
+
+  log_info "Disabling pushing to the upstream (parent) repository."
+  git remote set-url --push "${src_repo}" no_push
+
+  log_info "Searching and replacing repository name in source files."
+  find . \( ! -regex '.*/\.git/.*' \) -type f -exec \
+    perl -pi -e "s/${src_repo}/${dest_repo}/g" {} \;
+
+  log_info "Staging modified files."
+  git add --verbose .
+
+  log_info "Committing staged files to the develop branch."
+  git commit --message "Rename repository references after clone"
+
+  log_info "Creating the lineage.yml file."
+  cat << END_OF_LINE > .github/lineage.yml
+---
+lineage:
+  skeleton:
+    remote-url: https://github.com/${src_org}/${src_repo}.git
+version: '1'
+END_OF_LINE
+
+  log_info "Staging modified files."
+  git add --verbose .
+
+  log_info "Committing staged files to the develop branch."
+  git commit --message "Add lineage configuration"
+
+  log_info "Creating first-commits branch."
+  git checkout -b first-commits
+
+  remote_repo_status="unknown"
+  log_info "Checking for remote repository ${dest_org}/${dest_repo}."
+  remote_repo_name=$(gh repo view --json name --jq ".name" "${dest_org}/${dest_repo}") || :
+  if [[ -n ${remote_repo_name} ]]; then
+    log_ok "${dest_org}/${dest_repo} exists."
+    remote_repo_status="exists"
+  else
+    log_warn "${dest_org}/${dest_repo} does not yet exist."
+    log_info "Attempting to create a new remote repository."
+    # Note that gh will attempt to add the remote origin and will throw an error
+    # if it already exists.  We'll add the origin later if gh fails.
+    remote_repo_status="created"
+    gh repo create --public --confirm "${dest_org}/${dest_repo}" || remote_repo_status="failed"
+  fi
+
+  # Handle the different outcomes of the remote repository.
+  case $remote_repo_status in
+    created)
+      log_ok "The remote repository ${dest_org}/${dest_repo} was successfully created."
+      ;;&
+    existed | failed)
+      log_info "Adding a new remote origin for the repository."
+      # We can add the remote origin now since gh was unable to do it.
+      git remote add origin git@github.com:"${dest_org}/${dest_repo}".git
+      ;;&
+    created | existed)
+      log_info "Pushing develop and first-commit branches to the remote."
+      git push origin develop first-commits --set-upstream
+      log_info "Opening a new pull request for the first-commits branch."
+      log_info "If prompted, please select ${dest_org}/${dest_repo} as the base repository."
+      gh pr create --title "First commits" --assignee=@me --web
+      log_ok "Success!"
+      ;;
+    failed)
+      log_warn "The remote repository ${dest_org}/${dest_repo} counld not be created."
+      cat << END_OF_LINE
```

Comment:
Change tense of flag to match the previous changes

```suggestion
    created)
      log_ok "The remote repository ${dest_org}/${dest_repo} was successfully created."
      ;;&
    exists | failed)
      log_info "Adding a new remote origin for the repository."
      # We can add the remote origin now since gh was unable to do it.
      git remote add origin git@github.com:"${dest_org}/${dest_repo}".git
      ;;&
    created | exists)
      log_info "Pushing develop and first-commit branches to the remote."
      git push origin develop first-commits --set-upstream
      log_info "Opening a new pull request for the first-commits branch."
      log_info "If prompted, please select ${dest_org}/${dest_repo} as the base repository."
      gh pr create --title "First commits" --assignee=@me --web
      log_ok "Success!"
      ;;
    failed)
      log_warn "The remote repository ${dest_org}/${dest_repo} counld not be created."
      cat << END_OF_LINE
```